ingestion/src/metadata/profiler/interface/nosql/profiler_interface.py

```diff
@@ -193,7 +193,9 @@ def get_all_metrics(
             self.service_connection_config.__class__.__name__, self.connection
         )
         metric_list = [
-            self.compute_metrics(runner, metric_func) for metric_func in metric_funcs
+            self.compute_metrics(runner, metric_func)
+            for metric_func in metric_funcs
+            if metric_func.metrics
```

Comment:
I suggest moving this confition to [metric_filter.py](https://github.com/open-metadata/OpenMetadata/blob/8bacb44a715be2c6eacf5980c63bfa24095130a4/ingestion/src/metadata/profiler/processor/metric_filter.py).